app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -102,9 +177,9 @@ export default Controller.extend(ModalFunctionality, {
     return REMINDER_TYPES;
   },
 
-  @discourseComputed()
-  showLastCustom() {
-    return this.lastCustomReminderTime && this.lastCustomReminderDate;
+  @discourseComputed("lastCustomReminderTime", "lastCustomReminderDate")
```

Comment:
could probably use http://api.emberjs.com/ember/3.15/functions/@ember%2Fobject%2Fcomputed/and here